ci/travis/before_install_linux

```diff
@@ -2,4 +2,14 @@
 set -e
 
 sudo apt-get update
-sudo apt-get install libmatio-dev cppcheck astyle
+sudo apt-get install libmatio-dev cppcheck 
+
+if [ -n "$ASTYLE_FORMAT"]
+then
+    wget "https://sourceforge.net/projects/astyle/files/astyle/astyle 3.1/astyle_3.1_linux.tar.gz"
+    tar -xzf astyle_3.1_linux.tar.gz
+    cd ./astyle/build/gcc
+    make
+    sudo make install
```

Comment:
i wouldn't install it on the system;
maybe just append to the PATH env var, the path to the `astyle` executable 
that way, this can be run locally